src/textual/_time.py

```diff
@@ -42,3 +42,26 @@ async def sleep(secs: float) -> None:
         sleep_for = secs - 0.0005
         if sleep_for > 0:
             await asyncio_sleep(sleep_for)
+
+
+def get_time_no_wait() -> float:
+    """Get the current wall clock (monotonic) time.
+
+    Returns:
+        The value (in fractional seconds) of a monotonic clock,
+        i.e. a clock that cannot go backwards.
+    """
+    return time()
+
+
+async def get_time() -> float:
```

Comment:
I don't think we will need an `async` time. It was an artifact of the mockable clock idea I had.

Reply:
The tests seem to agree with you.
I saw it being used in `timer.py` and I thought it was needed, but it works just fine with the synchronous version as well.